dhis-2/dhis-test-web-api/src/test/resources/program/create_program.json

```
@@ -66,7 +66,14 @@
             "id": "PrZMWi7rBga"
           }
         }
-      ]
+      ],
+      "enrollmentDateLabel": "enrollmetdatelabel",
```

```suggestion
      "enrollmentDateLabel": "enrollmentdatelabel",
```

Personally I would use a more "real" value we would see in a UI like `Enrollment Date`. As with `"enrollmentdatelabel"` I thought this is a DB column 😅